src/styles/ada_ada_ada.js

```diff
@@ -60,9 +60,9 @@ let word = 'SILTA'
 const palette = fettepalette.generateRandomColorRamp({
   total: 9,
   centerHue: fxrand() * 360,
-  hueCycle: 0.3,
+  hueCycle: fxrand() * 0.6,
```

Comment:
please use this._p5.random() instead of fxrand() to be sure to be determistic

Reply:
@camilleroux Done.

Reply:
Thanks :)